source/js/common/bind-event-handlers.js

```diff
@@ -0,0 +1,25 @@
+import ReactGA from "../react-ga-proxy.js";
+
+/**
+ * Bind click handler to #donate-footer-btn
+ */
+const bindDonateFooterButtonHandler = () => {
```

Comment:
If this is the only thing it does right now, this probably needs a rename to at least `template-js-bindings` or something, but possibly even `./common/template-js-handles/donate-footer-button.js`